dataplane/linux/vxlan_mgr.go

```diff
@@ -175,7 +240,8 @@ func (m *vxlanManager) CompleteDeferredWork() error {
 
 	if m.routesDirty {
 		// Iterate through all of our L3 routes and send them through to the route table.
-		var routes []routetable.Target
+		var vxlanRoutes []routetable.Target
+		var parentRoutes []routetable.Target
```

Comment:
nit on naming - rather than calling these "parent routes" I'd suggest something like "noEncapRoutes" - I think that will be more clear to readers of this code and more accurately describes what the routes are.

Similarly for other places you're using "parent" right now - e.g., `parentRouteTable` is probably better called `noEncapRouteTable`

Reply:
Or even better you could call them `defaultRoutes` since that aligns with the route type.

Reply:
I'll update that.